nservicebus/messaging/forwarding.md

```diff
@@ -7,7 +7,18 @@ redirects:
 ---
 
 
-Use this feature to forward successfully processed messages from an endpoint to a specified destination endpoint. If there is a need to collect all successfully processed messages from all endpoints, then use the [Auditing Feature](/nservicebus/operations/auditing.md) instead.
+Use this feature to forward successfully processed messages from an endpoint to a specified destination endpoint. Forwarding messages is particularly useful in complex upgrade scenarios, when the old version and new version of a particular endpoint are running side-by-side.
+
+Note that forwarding is a solution to a specific problem and generally should be used for a limited transition period. In case of long-term need to duplicate messages the [Publish-Subscribe](/nservicebus/messaging/publish-subscribe/) pattern will be usually more appropriate.
```

Comment:
>Note that forwarding is a solution to a specific problem and generally should be used for a limited transition period.

I am not sure if this is true. You can use forwarding when you have a workflow where after a message is processed by endpoint 1, you want endpoint 2 to do more processing. Sort of like an itinerary pattern.  I think the original use case was meant for this.